### PR TITLE
Pf 5 fix es lint warnings

### PR DESCRIPTION
**Pull request by ZheleznovTN**
@dogi @lmmrssa check and merge after PF-2 will be merged in main branch.

### CODE REVIEW

**Review by lmmrssa — CHANGES_REQUESTED** [review]
This is too big changes PR to review. And also most of changes seems to be just **quote** and **semi-colon** So it will better if we separate these changes and actual file or code changes to different PR.
Also we had implemented standard linter which you can test from `npx standard --fix`, so this lint change will touch almost all files. So lets just make lint changes first if we want to do that and also fix the standard linter to follow those guideline so our codeclimate can track accordingly.
And then create another PR for changes such as new button components. And please avoid adding empty files. Let's just add files as we have something to review on that file.
Basically we want to have PR will changes relevant to specific changes only so its possible to merge them separately and easier to know what to review.
@ZheleznovTN sorry for the long comment but right now looking at 114 files and figuring out what actually got changed is difficult to keep track of and lot of chances to miss the actually important code block

**Comment by ZheleznovTN** [comment]
@lmmrssa this pr only contains eslint fixes, for example unused variables and maybe auto code style fixes by eslint and prettier. So this PR don't contain any of logic or functionality changes.

**Comment by ZheleznovTN** [comment]
@lmmrssa and if you merge PF-2 PR in master - this PR will reduce size

**Comment by lmmrssa** [comment]
comment added is for PF-2 as well its better to have one linting PR and another for files . right now there are new files also in PF-2.
Also as you are  changing linting please fix lint test guideline as well according to linting format you are using. Else there will be conflict between lint test and lint applied

**Comment by ZheleznovTN** [comment]
@lmmrssa where you have eslint config in project? i don't see that

**Comment by lmmrssa** [comment]
we are using standard package to fix linting and as we were using its default rules we had not setup any custom guideline. and new  lint are different so we need to specify those guidelines so that standard can use it. npx standard --fix does the autolinting. and look at standardjs.com on setting custom rules
